internal/api/registry/manifests.go

```diff
@@ -72,7 +72,16 @@ func (a *API) handleGetOrHeadManifest(w http.ResponseWriter, r *http.Request) {
 				}
 			}
 
-			dbManifest, manifestBytes, err = a.processor().ReplicateManifest(r.Context(), *account, *repo, reference, keppel.AuditContext{
+			tagPoliciesStr, err := a.db.SelectStr(api.GetTagPolicyByAccountName, account.Name)
+			if respondWithError(w, r, err) {
+				return
+			}
+			tagPolicies, err := keppel.ParseTagPolicies(tagPoliciesStr)
+			if respondWithError(w, r, err) {
+				return
+			}
```

Comment:
Since we have the query in `internal/api/utils.go` already, can we put all of this (take account name, pull JSON from DB and parse) as a helper function there?